timechart/displays/main_display.py

```diff
@@ -402,7 +401,13 @@ def __init__(self, parent=None, args=[], macros=None, show_pv_add_panel=True, co
         # If there is an imported config file, let's start TimeChart with the imported configuration data
         if config_file:
             importer = SettingsImporter(self)
-            importer.import_settings(config_file)
+            try:
+                importer.import_settings(config_file)
+            except SettingsImporterException as error:
+                display_message_box(QMessageBox.Critical, "Import Failure",
+                                    "Cannot import the file '{0}'. Check the log for the error details."
+                                    .format(config_file))
+                logger.error("Cannot import the file '{0}'. Error: {1}".format(config_file, error))
```

Comment:
You can use `logger.exception` in here and skip sending the `Error: ` part since it will be automatically added to the logger with the proper stack trace.

Reply:
Done.